stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/event/EventHandlerImpl.kt

```diff
@@ -389,9 +389,11 @@ internal class EventHandlerImpl(
             when (event) {
                 is NotificationChannelTruncatedEvent -> {
                     domainImpl.repos.deleteChannelMessagesBefore(event.cid, event.createdAt)
+                    domainImpl.repos.evictChannel(event.cid)
                 }
                 is ChannelTruncatedEvent -> {
                     domainImpl.repos.deleteChannelMessagesBefore(event.cid, event.createdAt)
+                    domainImpl.repos.evictChannel(event.cid)
```

Comment:
Can we move this logic into `RepositoryFacade??
Whenever we "delete channels messages before a given date", they should be removed from the local cache as well. If we keep it as two different methods, every time we want to perform the first action, we need to remember that the second action need to be done as well.